nucypher/control/controllers.py

```diff
@@ -302,8 +304,22 @@ def start(self,
     def __call__(self, *args, **kwargs):
         return self.handle_request(*args, **kwargs)
 
-    def handle_request(self, method_name, control_request, *args, **kwargs) -> Response:
+    @staticmethod
+    def json_response_from_worker_pool_exception(exception):
+        json_response = {}
+        json_response['failure_message'] = f"{exception}"
```

Comment:
Why mutate here? IMO, initializing the dictionary with the first value is simple and direct.

Reply:
✔️